invenio/modules/records/api.py

```diff
@@ -58,11 +60,30 @@ def create_many(cls, blobs, master_format, **kwargs):
         raise NotImplementedError()
 
     @classmethod
-    def get_record(cls, recid, reset_cache=False):
+    def get_record(cls, recid, reset_cache=False, revision=None):
         """Get one record from the DB.
 
         :param reset_cache: If set to `True` it creates the JSON again.
+        :param revision: If not None and revision exists it tries to retrieve
+            it from `history_record_json`.
         """
+        from invenio.ext.sqlalchemy import db
```

Comment:
Ah, sorry just saw that you just moved an existing import. Anyway, perhaps it still works if the import is in the top.